content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -90,30 +90,30 @@ for database debugging.
 
 ## Forward a local port to a port on the pod
 
-1.  `kubectl port-forward` allows using resource name, such as a service name, to select a matching pod to port forward to since Kubernetes v1.10.
+1.  `kubectl port-forward` allows using resource name, such as a service name, to select a matching pod to port forward to since Kubernetes v1.10. Example for using port forwarding to extend port 80 to 8081 as follows
```

Comment:
I already know what this does; however, the term “extend” here doesn't make sense to me.
1. How about rewording to explain that someone could connect a client to port 8081 on localhost and access port 80 on the Redis master.

2. Does Redis listen on port 80?

Reply:
Hi @sftim I think for redis is by default to 6379.  I think it would be great to rewording it rather than I did by saying ' extend'. Here my point is really about how to make beginners be clear about the usage of the port-forward command. Thanks for bring your thoughts up. Much appreciated!